### PR TITLE
Use tumor_bam and normal_bam subroutines instead of RefAlign builds

### LINKED ISSUE #1409: Add Somatic Validation tracks to DumpIgvXml ClinSeq command.
The DumpIgvXml command relies on finding the tumor_build and normal_build which only exist as ReferenceAlignment builds on Somatic Variation inputs.  The DumpIgvXml command should resolve the path to the tumor/normal bed/bam file for all somatic model types.

### PR DESCRIPTION
**Pull request by susannasiebert**
Closes #1409

### CODE REVIEW

**Comment by jasonwalker80** [comment]
+1

File: lib/perl/Genome/Model/ClinSeq/Command/DumpIgvXml.pm [hunk 1]
```diff
@@ -194,79 +194,76 @@ sub execute {
...
-                );
-                my $wgs_normal_xml = $wgs_normal_track->{xml};
-                $wgs_normal_xml = $self->generate_panel_xml(
-                    '-track_xml'   => $wgs_normal_xml,
-                    '-panel_count' => $panel_count++
-                );
-                push(@resources, $wgs_normal_track->{resource});
-                $main_features_track_xml .= $wgs_normal_xml;
-            }
+            my $wgs_normal_build = $wgs_build->can('normal_build') ? $wgs_build->normal_build : $wgs_build;
```
Comment (tmooney) [inline lib/perl/Genome/Model/ClinSeq/Command/DumpIgvXml.pm]: Would this be something useful to add to the `Genome::Model::Build::SomaticInterface`? Something like `build_for_normal_bam`?

(I was surprised to see that `tumor_bam` and `normal_bam` aren't really a part of that interface.)

**Comment by tmooney** [comment]
+1 🔥

File: lib/perl/Genome/Model/ClinSeq/Command/DumpIgvXml.pm (hunk 1 above)
Comment (susannasiebert) [inline lib/perl/Genome/Model/ClinSeq/Command/DumpIgvXml.pm]: Great idea. I pushed a commit to implement your suggestion.

**Comment by jasonwalker80** [comment]
+1, I like the suggestion of moving these into the interface.

**Comment by tmooney** [comment]
(+1 again!)

**Pull request merged**